### PR TITLE
SA .357 Custom Item Addition

### PR DESCRIPTION
**Pull request by Max-023**
Adds a new loadout option in gear preferences for the SA .357 Spearhead. Provides a guncase with ammunition for it, with potential for the player to scavenge more ammunition in the field from the standard CMB Spearhead. Also changes the custom M44 to come in a gun case with marksman ammunition and adjusts pistol belts to include the new addition..

### CODE REVIEW

File: code/modules/projectiles/guns/revolvers.dm [hunk 1]
```diff
@@ -746,3 +746,14 @@
 
 /obj/item/weapon/gun/revolver/cmb/normalpoint
 	current_mag = /obj/item/ammo_magazine/internal/revolver/cmb
+
+//-------------------------------------------------------
+//MARINE REVOLVER //Spearhead exists in Alien cannon, this one is based on the Dark Descent model.
+
+/obj/item/weapon/gun/revolver/cmb/marine
+	name = "\improper SA .357 Revolver"
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/revolvers.dm:754]:
```suggestion
	name = "\improper SA .357 revolver"
```
i'd still decapitalize the "R" in its name.

**Review by AndroBetel — CHANGES_REQUESTED** [review]
these should be fixed

`The M276 pattern M44 holster rig cannot hold the SA .357 Revolver.`
image

**Comment by Max-023** [comment]
Alright, that should fix all the issues.

**Review by morrowwolf — CHANGES_REQUESTED** [review]
I feel I misunderstand, what is the reasoning for the switch?

**Comment by Max-023** [comment]
The swap is based on the fact that the Spearhead is shown to be the canonical revolver for the USCM, based on the game Aliens: Dark Descent. So I've just gone in and swapped the M44 from the vendors with a new Spearhead, as well as updating the CMB to look better.

File: code/game/objects/items/storage/belt.dm [hunk 2]
```diff
@@ -1232,13 +1232,15 @@
 			"icon_y" = -5))
 
 /obj/item/storage/belt/gun/m44
-	name = "\improper M276 pattern M44 holster rig"
-	desc = "The M276 is the standard load-bearing equipment of the USCM. It consists of a modular belt with various clips. This version is for the M44 magnum revolver, along with six small pouches for speedloaders. It smells faintly of hay."
+	name = "\improper M276 pattern revolver pistol holster rig"
```
Comment (AndroBetel) [inline code/game/objects/items/storage/belt.dm:1235]: should be just "M276 pattern revolver holster rig"

File: code/modules/cm_marines/equipment/guncases.dm [hunk 3]
```diff
@@ -388,6 +388,23 @@
...
+
+/obj/item/storage/box/guncase/m44/fill_preset_inventory()
+	new /obj/item/attachable/flashlight(src)
+	new /obj/item/weapon/gun/revolver/m44/custom/mp(src)
+	new /obj/item/ammo_magazine/revolver/marksman(src)
+	new /obj/item/ammo_magazine/revolver/marksman(src)
+	new /obj/item/ammo_magazine/revolver/marksman(src)
+	new /obj/item/ammo_magazine/revolver/marksman(src)
+	new /obj/item/ammo_magazine/revolver/marksman(src)
+	new /obj/item/ammo_magazine/revolver/marksman(src)
```
Comment (AndroBetel) [inline code/modules/cm_marines/equipment/guncases.dm:406]: that's a lot of non-standard ammo for apparently no reason, and also a flashlight? dont think thats necessary

File: code/modules/cm_marines/equipment/guncases.dm [hunk 4]
```diff
@@ -401,3 +418,20 @@
...
+
+/obj/item/storage/box/guncase/cmb/fill_preset_inventory()
+	new /obj/item/attachable/flashlight(src)
+	new /obj/item/weapon/gun/revolver/cmb/marine(src)
+	new /obj/item/ammo_magazine/revolver/cmb/normalpoint(src)
+	new /obj/item/ammo_magazine/revolver/cmb/normalpoint(src)
+	new /obj/item/ammo_magazine/revolver/cmb/normalpoint(src)
+	new /obj/item/ammo_magazine/revolver/cmb/normalpoint(src)
+	new /obj/item/ammo_magazine/revolver/cmb/normalpoint(src)
+	new /obj/item/ammo_magazine/revolver/cmb/normalpoint(src)
```
Comment (AndroBetel) [inline code/modules/cm_marines/equipment/guncases.dm:437]: dont think custom revolver needs that many speedloaders and a flashlight, make it 2 just like with 1911

besides, you can still scavenge ammo from CMBs and security groundside, i think that would be interesting

File: code/modules/projectiles/guns/revolvers.dm [hunk 5]
```diff
@@ -347,6 +347,9 @@
 	icon_state = "m44rc"
 	item_state = "m44rc"
 
+/obj/item/weapon/gun/revolver/m44/custom/mp //loadout w/ ammo
+	current_mag = /obj/item/ammo_magazine/internal/revolver/m44/marksman
```
Comment (AndroBetel) [inline code/modules/projectiles/guns/revolvers.dm:351]: dont think loadout equipment should have buffs.....

**Review by AndroBetel — CHANGES_REQUESTED** [review]
morrow still gotta approve of adding it to loadout, but i think its fine personally

**Comment by AndroBetel** [comment]
stale for more than a month, author seems to be inactive

**Pull request closed without merging**